src/oom/oomd-util.c

```diff
@@ -122,6 +122,16 @@ uint64_t oomd_pgscan_rate(const OomdCGroupContext *c) {
         return c->pgscan - last_pgscan;
 }
 
+bool oomd_mem_free_below(const OomdSystemContext *ctx, int threshold_permyriad) {
+        uint64_t mem_threshold;
+
+        assert(ctx);
+        assert(threshold_permyriad <= 10000);
+
+        mem_threshold = ctx->mem_total * threshold_permyriad / (uint64_t) 10000;
+        return (ctx->mem_total - ctx->mem_used) < mem_threshold;
```

Comment:
just to be safe, maybe use LESS_BY() for the the subtraction? (i.e. saturated subtraction)?

i mean, the kernel i think doesn't guarantee us that mem_total and mem_used are consisten with each other